app/models/telegram_bot.rb

```diff
@@ -18,10 +19,17 @@ def post_webhook_to_telegram
     telegram_bot = Telegram::Bot::Client.new(token: token, username: username)
 
     begin
-      request = telegram_bot.set_webhook(url: ENV["TELEGRAM_CALLBACK_URL"])
+      request = telegram_bot.set_webhook(url: telegram_webhook_url)
```

Comment:
@radinreth please use only one telegram_webhook_url where it could be updated between Development & Production via the environment variable